backend/src/main/java/com/benine/backend/http/AutoPresetCreationHandler.java

```diff
@@ -54,10 +55,15 @@ public void handle(String s, Request request, HttpServletRequest httpServletRequ
     IPCamera ipcam = (IPCamera) cam;
    
     try {
-      Collection<IPCameraPreset> presets = creator.createPresets(ipcam, creator.generateSubViews());
-      
+      ArrayList<IPCameraPreset> presets =
+              new ArrayList<>(creator.createPresets(ipcam, creator.generateSubViews()));
+
+      String name = request.getParameter("name");
       JSONObject jsonObject = new JSONObject();
       JSONArray idsJson = new JSONArray();
+      for (int i = 0; i < presets.size(); i++) {
+        presets.get(i).setName(name + i);
```

Comment:
The preset names will not be updated in the database, you need to call the updatePreset method.